scripts/generateDiffData/index.test.ts

```diff
@@ -0,0 +1,425 @@
+/*
```

Comment:
👍  Thank you for the test! :) Should we add this to the jenkinsfile for test during CI?

Reply:
@hlminh2000 yes, great idea. Will do in different PR due to current Jenkins issues.